sdk/agent_diff/client.py

```
@@ -0,0 +1,134 @@
+        return TestSuiteDetail.model_validate(response.json())
+
+    def get_test(self, test_id: UUID):
+        response = requests.get(
+            f"{self.base_url}/api/platform/tests/{test_id}",
+            headers={"X-API-Key": self.api_key},
+            timeout=5,
+        )
+        response.raise_for_status()
+        return response.json()
+
+    def create_test(self, test, testSuiteId: UUID):
```

Ruff (0.14.1)

Unused method argument: `testSuiteId`

(ARG002)